spec/models/piece_spec.rb

```diff
@@ -1,5 +1,67 @@
 require 'rails_helper'
 
 RSpec.describe Piece, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'obstructed method' do
+    it 'should return nil if invalid move' do
+      game = FactoryGirl.create(:game)
+      piece = FactoryGirl.create(:piece, x_position: 4, y_position: 4, type: 'Rook', game: game)
+      expect(piece.obstructed?(1, 3)).to eq nil
```

Comment:
Reasoning: we'll let another method handle if the move is on the board or not. :)